### DISCUSSION TITLE
error is coming while connecting catalouge module and i already updated catalouge server ip add in front end

### DISCUSSION THREAD

**Discussion by sreekanthacchu9** [learndevopsonline/learndevopsonline#1682] [Q&A] [answered]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/270dd9ff-10d2-43db-aefd-5784b93631ae)

**Comment by sreedharm07** [comment]
ping your ip here i will check

**Reply by sreekanthacchu9** [reply]
this is my front end nginx server public ip 54.227.93.139   
this is mongdb public ip 35.173.239.20
this is catalouge pubplic ip 54.242.217.173


still getting same error

**Reply by sreekanthacchu9** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/49873858-5321-4a3d-a13d-a154bb2b13c8)

**Reply by sreedharm07** [reply]
here catalogue word in last missing and <> symbols should not use there correct your catalogue service file

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/f1d37ecf-ef76-4849-93f4-6396009af24b)

**Reply by sreekanthacchu9** [reply]
I added catalouge but still getting same error and i reloaded the systemd configuration file aslo and restarted nginx also.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/01461b67-2847-42a6-8608-cfb30d8396a7)

**Reply by sreedharm07** [reply]
i suggested before to remove <> symbols between ip
but i see there is still that symbols are there remove that

**Reply by sreekanthacchu9** [reply]
yes,i removed symbols but same error coming
![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/293c2847-31eb-44a5-bf2b-0a4ed1164408)

**Reply by sreedharm07** [reply]
its working
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/a072783a-d197-4e1b-9ef6-787bc3afb137)

**Reply by sreekanthacchu9** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/006c4fa0-8d4d-410a-978c-e26eb435b0ce)
could you check it once ,whenever i gave credentials it is showing like this.

**Reply by sreedharm07** [reply]
to work credentials you should configure user component 
you only configured until catalogue

**Reply by sreekanthacchu9** [reply]
OK Thanks sree

**Reply by sreekanthacchu9** [reply, marked as answer]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/1ebebfda-228f-4502-bc8a-f85db1784035)
thanks for your support @sreedharm07 
finally came .
